make/testing_flags.mk

```diff
@@ -54,6 +54,11 @@ ifeq ($(dbbackend),postgres)
 DEV_TAGS += test_db_postgres
 endif
 
+# Run universe tests with increased scale for performance testing.
+ifneq ($(long-tests),)
+TEST_FLAGS += long-tests
```

Comment:
NOTE: Unsure if this is the correct approach here.

Reply:
Sorry, I think I lead you on the wrong path here... Turns out flags aren't as easy to use as build tags.
Have a working version here, feel free to just apply this to your commit directly: https://github.com/guggero/taproot-assets/commit/34a0a2824e8869d9390ea8913761f26df937fe57

Can test with: `make unit pkg=tapdb` vs. `make unit pkg=tapdb long-tests=true`.